frontend/src/components/program/ScheduleEntryFilters.vue

```diff
@@ -0,0 +1,254 @@
+<template>
+  <div class="d-flex flex-wrap items-baseline" style="overflow-y: auto; gap: 10px">
+    <BooleanFilter
+      v-if="loadingEndpoints !== true && loadingEndpoints.campCollaborations !== true"
+      v-model="showOnlyMyActivities"
+      :label="$tc('components.program.scheduleEntryFilters.onlyMyActivities')"
```

Comment:
Suggestion: conditional label "Nur meine Aktivitäten/Only my activities" (if no other filters set) or "Alle meine Aktivitäten/All my activities" (if other filters are applied) to make it clear that all other filters will be reset.

Reply:
#4956